static/manual.Rmd

```diff
@@ -469,9 +469,70 @@ We plan to add more documentation to ease the creation of locally specific scena
 
 </div>
 
-<!-- [8\. How can the PCT be used to build a business case for cycling?](javascript:toggleElement('a8'))
+[8\. Why is there no increase in cycling under Go Dutch?](javascript:toggleElement('a8'))
```

Comment:
See this as an edge case, not sure it warrants a section in the Manual. One for offline discussion.